docs/en/sql-reference/table-functions/azureBlobStorage.md

```diff
@@ -85,6 +86,7 @@ SELECT count(*) FROM azureBlobStorage('DefaultEndpointsProtocol=https;AccountNam
 ## Hive-style partitioning {#hive-style-partitioning}
 
 <ExperimentalBadge/>
+<CloudNotSupportedBadge/>
```

Comment:
It is supported perfectly.

Reply:
Sure but then it needs to be moved out of experimental, beta min. We explicitly
don't support experimental features and this would then be an exception. @melvynator

Reply:
https://github.com/ClickHouse/ClickHouse/blob/321d3f10bf22a7644b4ee9ca92e85252a71bce2b/src/Core/Settings.cpp#L5935C4-L5937C19

In the cloud all experimental features should be disabled. We then need to either, remove the experimental label or rename the setting to highlight it is experimental.